akka-cluster/src/main/resources/reference.conf

```diff
@@ -33,7 +33,12 @@ akka {
     # partition are created in surviving partition. The purpose of this margin is that 
```

Comment:
in the above sentence (L32) it says "until" but is that correct? Isn't it "after"?